package.json

```diff
@@ -24,11 +24,17 @@
     "lint:fix": "npm run lint -- --fix",
     "test:unit": "cross-env NODE_ENV=development jest -c jest.config.unit.ts",
     "test:e2e": "cross-env NODE_ENV=development jest -c jest.config.e2e.ts",
+    "test:e2e:dev:backend": "npm run test:e2e:dev --workspace @dddforum/backend",
+    "test:e2e:dev:frontend": "npm run test:e2e:dev --workspace @dddforum/frontend",
     "test:infra": "cross-env NODE_ENV=development jest -c jest.config.infra.ts"
   },
+  "dependencies": {
+    "axios": "^1.4.0"
```

Comment:
Since this is a prod dependency it needs to be installed in every package where it's used.

Reply:
I've fixed it in https://github.com/stemmlerjs/dddforumv2/pull/9#issuecomment-1669770836.